### PR TITLE
Booking show page and Dashboard

### LINKED ISSUE #16: Renter Dashboard - Upcoming Bookings link to Booking Show Page
As a Renter, when I visit the Dashboard, I want to see my upcoming bookings, so that I can click a link to view the booking. Then I am redirected to the booking show page.

- Create bookings table that has a status column

### LINKED ISSUE #19: Renter Dashboard - Pending Bookings link to the Booking Show Page
As a Renter, when I visit the Dashboard, I want to see my pending bookings, so that I can click a link to view the pending booking. Then I am redirected to the booking show page.

### LINKED ISSUE #21: Host Yard Booking Show Page
As a host
When I visit a booking
I see the following:

- [x] yard name
- [x] address
- [x] date
- [x] ime/duration
- [x] description of the booking
- [x] total cost of booking
- [ ] button to cancel the booking
- [x] booking status (pending, rejected, or approved)
- [ ] renter’s information
- [ ] host information

### PR DESCRIPTION
**Pull request by jbranham1**
Added booking show page
Added a little more styling
Update dashboards to look more alike
Added approve and reject to host bookings on the dashboard
Refactored booking facade
Added delete booking service.
Close #21
Close #16
Close #19

To-do: delete booking if it is not pithing 48 hours.

### CODE REVIEW

**Review by domo2192 — APPROVED** [review]
Looks great!!!!

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,6 +1,15 @@
 class BookingsController < ApplicationController
   before_action :set_yard, only: [:new, :create]
 
+  def show
+    @booking = BookingFacade.get_booking(params[:id])
+
+    # if @booking.empty?
+    #   flash[:error] = 'Data cannot be accessed at this time'
+    #   redirect_to host_dashboard_index_path
+    # end
```
Comment (abreaux26) [inline app/controllers/bookings_controller.rb]: When you get a chance, do you mind removing this?

**Review by Gvieve — APPROVED** [review]
This looks great! Thanks so much for tackling all of it.

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (jbranham1) [reply app/controllers/bookings_controller.rb]: Done!

**Pull request merged**